modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemsXlsMVCResourceCommand.java

```diff
@@ -0,0 +1,462 @@
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow, LanguageUtil.get(locale, "author"));
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow, LanguageUtil.get(locale, "type"));
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow, LanguageUtil.get(locale, "subtype"));
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow,
+			LanguageUtil.get(locale, "site-or-asset-library"));
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow, LanguageUtil.get(locale, "status"));
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow,
+			LanguageUtil.get(locale, "categories"));
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow, LanguageUtil.get(locale, "tags"));
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow,
+			LanguageUtil.get(locale, "modified-date"));
+
+		return headerRowCellIndex;
+	}
+
+	private int _createCell(int cellIndex, Row row, String value) {
+		Cell titleDataCell = row.createCell(cellIndex++);
+
+		titleDataCell.setCellValue(value);
+
+		return cellIndex;
+	}
+
+	private CellStyle _createCellStyle(
+		boolean bold, String fontName, short heightInPoints,
+		Workbook workbook) {
+
+		Font font = workbook.createFont();
+
+		font.setBold(bold);
+		font.setFontHeightInPoints(heightInPoints);
+		font.setFontName(fontName);
+
+		CellStyle style = workbook.createCellStyle();
+
+		style.setFont(font);
+
+		return style;
+	}
+
+	private void _createDataRow(
+		ResourceRequest resourceRequest, ResourceResponse resourceResponse,
+		Locale locale, ContentDashboardItem<?> contentDashboardItem, Row row,
+		ThemeDisplay themeDisplay) {
+
+		int cellIndex = _createBasicDataCells(
+			0, locale, row, contentDashboardItem);
+
+		if (contentDashboardItem instanceof FileEntryContentDashboardItem) {
+			cellIndex = _createFileSpecificDataCells(
+				cellIndex, locale, resourceResponse, resourceRequest, row,
+				themeDisplay, contentDashboardItem);
+		}
+
+		if (contentDashboardItem instanceof
+				JournalArticleContentDashboardItem) {
+
+			_createJournalArticleSpecificDataCells(
+				cellIndex + _FIRST_JOURNAL_SPECIFIC_FIELD_INDEX, locale,
+				resourceResponse, resourceRequest, row, themeDisplay,
+				contentDashboardItem);
+		}
+	}
+
+	private int _createFileHeaderCells(
+		Row headerRow, int headerRowCellIndex, Locale locale) {
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow,
+			LanguageUtil.get(locale, "description"));
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow,
+			LanguageUtil.get(locale, "extension"));
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow,
+			LanguageUtil.get(locale, "file-name"));
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow, LanguageUtil.get(locale, "size"));
+
+		return headerRowCellIndex;
+	}
+
+	private int _createFileSpecificDataCells(
```

Comment:
Call it `getSpecificInformationDataCells` to match the name of the method from the contentDashboardItem.